## WO-DEVOPS-004K note
The onboarding docs and pre-commit docs-only scope guard were pushed with one-time operator-authorized local hook bypass.
Reason: local hooks are currently blocked by existing local package/tooling issues unrelated to this docs-only change:
1. Pre-commit token gate originally failed because `pnpm-lock.yaml` lacked integrity metadata for the SheetJS `xlsx` tarball.
2. After a local narrow repair, the gate advanced to `ERR_PNPM_EXOTIC_SUBDEP` from `packages/terrabuild -> aframe@1.7.1 -> three-bmfont-text`.
3. `pnpm exec prettier` also triggers workspace dependency resolution and hits the same package-governance blocker.
4. The clean worktree has no local `node_modules/.bin/prettier`.
5. The pre-push hook runs `TF_STRICT_PUSH=1 npm run test:unit`, but `vitest` is not available on PATH.
This commit remains docs/tooling-scope only:
- `.husky/pre-commit`
- `docs/onboarding/DEV_SETUP.md`
- `docs/onboarding/AGENT_START_HERE.md`
- `docs/onboarding/TROUBLESHOOTING.md`
Not included:
- `pnpm-lock.yaml`
- dependency changes
- package.json changes
- runtime code
- pipeline YAML
- GitHub workflows
- Docker/Helm/Kubernetes
- secrets/county/PACS data
Follow-up required:
- package-governance WO for `pnpm-lock.yaml` / SheetJS integrity / `terrabuild` exotic subdependency
- local tooling WO to make pre-commit/pre-push usable in clean worktrees

Do not mark PR ready.
Do not merge.

--- a/.husky/pre-commit
+++ b/.husky/pre-commit
@@ -95,7 +95,7 @@ STAGED_FILES="$(git diff --cached --name-only 2>/dev/null || true)"
 RUN_UI_TOKEN_GATE=0
 
 if [ -n "$STAGED_FILES" ]; then
-  TOKEN_RELEVANT_FILES="$(echo "$STAGED_FILES" | grep -Ev '^(docs/|[^/]+\.md$|\.husky/pre-commit$)' || true)"
+  TOKEN_RELEVANT_FILES="$(echo "$STAGED_FILES" | grep -Ev '^(docs/|\.md$|\.husky/pre-commit$)' || true)"
   if echo "$TOKEN_RELEVANT_FILES" | grep -Eq '^(frontend/|apps/|packages/|os-platform/|tools/|scripts/|src/|components/|styles/|tokens/|themes/|package\.json$|pnpm-lock\.yaml$|pnpm-workspace\.yaml$|\.npmrc$|\.husky/)'; then
     RUN_UI_TOKEN_GATE=1
   fi

